dbms/src/Core/SortDescription.cpp

```diff
@@ -26,8 +26,7 @@ std::string SortColumnDescription::getID() const
 {
     WriteBufferFromOwnString out;
     out << column_name << ", " << column_number << ", " << direction << ", " << nulls_direction;
-    if (collator)
-        out << ", collation locale: " << collator->getLocale();
+    out << ", collation locale: " << collator->getCollatorId();
```

Comment:
I think we should keep the nullptr check?

Reply:
Yes :(